travis/run_pylint.py

```diff
@@ -84,10 +87,13 @@ def main(paths, config_file, sys_paths=None, extra_params=None):
     to check fails of odoo modules.
     If expected errors is equal to count fails found then
     this program exit with zero otherwise exit with counted fails"""
-    stats = run_pylint(
-        list(paths), config_file.name, sys_paths=sys_paths,
-        extra_params=extra_params)
-    count_fails = get_count_fails(stats)
+    try:
+        stats = run_pylint(
+            list(paths), config_file.name, sys_paths=sys_paths,
+            extra_params=extra_params)
+        count_fails = get_count_fails(stats)
+    except ValueError:
+        count_fails = -1
```

Comment:
this has to be `count_fails = 0`.
See https://travis-ci.org/OCA/contract/jobs/77930562